src/Http/Http.Abstractions/test/UseMiddlewareTest.cs

```diff
@@ -293,13 +379,27 @@ private class MiddlewareInjectInvokeNoService
         public Task Invoke(HttpContext context, object value) => Task.CompletedTask;
     }
 
+    private class MiddlewareKeyedInjectInvokeNoService
```

Comment:
Doesn't seem like this class is needed? Just use the other class and don't add services to DI?

Reply:
@BrennanConroy, hmm, appears to be the case. I'll double check it in the morning and adjust it accordingly.